### PR TITLE
One Click Fund Issue Link In Townsquare

### LINKED ISSUE #6149: One Click Fund Issue Link in Townsquare
### User Story

When I have a developer ready to work on an issue, I'd like a one click fund issue link as an option in the townsquare thread because this helps to streamline my workflow.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: No one click fund

### Current Behavior
No quick way to fund an issue 

### Expected Behavior
If thread contains github link, display ability to fund that issue via menu 
Open fund issue in new tab with url pre-populated in the form

### Definition of Done

- [ ] Anyone can one click fund an issue
- [ ] If thread contains github link, display ability to fund that issue via menu 
- [ ] Open fund issue in new tab with url pre-populated in the form
- [ ] Once the issue has been funded, show the comment in the original townsquare thread
- [ ] Post a video or screenshots for core team review

Screen Shot 2020-03-05 at 5 56 58 AM

### PR DESCRIPTION
**Pull request by zoek1**
##### Description

- [X] Anyone can one click fund an issue
- [X]  If thread contains github link, display ability to fund that issue via menu
image
- [X]  Open fund issue in new tab with url pre-populated in the form
DeepinScreenshot_20200305152705
- [X]  Once the issue has been funded, show the comment in the original townsquare thread
image

- [x]  Post a video or screenshots for core team review

##### Refers/Fixes
#6149

##### Testing

https://www.dropbox.com/s/1dkc5ueas1qydt2/2020-03-09%2005-48-08.flv?dl=0

### CODE REVIEW

File: app/assets/v2/js/activity.js [hunk 1]
```diff
@@ -707,6 +707,19 @@ $(document).ready(function() {
...
   });
 
+  $(document).on('click', '.fund_issue', function(e) {
+    e.preventDefault();
+    var url = $(this).data('url');
+
+    copyToClipboard(url);
+    _alert('Link copied to clipboard.', 'success', 1000);
+    $(this).addClass('open');
+    var $target = $(this);
```
Comment (thelostone-mc) [inline app/assets/v2/js/activity.js]: ^ const

File: app/assets/v2/js/activity.js [hunk 2]
```diff
@@ -707,6 +707,19 @@ $(document).ready(function() {
     }, 300);
   });
 
+  $(document).on('click', '.fund_issue', function(e) {
+    e.preventDefault();
+    var url = $(this).data('url');
```
Comment (thelostone-mc) [inline app/assets/v2/js/activity.js]: const

File: app/assets/v2/js/pages/new_bounty.js [hunk 3]
```diff
@@ -597,6 +597,7 @@ $('#submitBounty').validate({
       */
       createBounty(data);
     } else {
+      console.log(data)
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/new_bounty.js]: let's remove this

File: app/assets/v2/js/pages/create_bounty/token.js [hunk 4]
```diff
@@ -75,7 +75,8 @@ createBounty = data => {
     'attached_job_description': hiring.jobDescription,
     'eventTag': metadata.eventTag,
     'auto_approve_workers': data.auto_approve_workers ? 'True' : 'False',
-    'web3_type': 'qr'
+    'web3_type': 'qr',
+    'activity': data.activity
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/create_bounty/token.js]: Ah nice you've also covered for ETC

**Comment by zoek1** [comment]
@danlipert @thelostone-mc could i get your feedback on this please?  :smile:

**Comment by owocki** [comment]
did this go live? i dont see it on my post https://bits.owocki.com/ApujKEv9

**Comment by danlipert** [comment]
@owocki its live: 
Screen Shot 2020-04-16 at 12 06 02 AM

**Pull request merged**